source/edge-server/configure.txt

```diff
@@ -534,15 +576,15 @@ Sync All Documents
 ```````````````````
 
 You can sync all documents between Edge Server instance and Atlas. This uses 
-the ``truepredicate`` or ``"*"`` query. Because this is a simple query, you 
-can pass it as a string with the ``edgectl --query`` argument.
+the ``truepredicate`` or ``"*"`` query. Because this is a global query, you 
+can pass it as a string with the ``edgectl --global-query`` argument.
```

Comment:
this happens by default, the global query is truepredicate when its not passed

Reply:
Yeah, I noted that above but probably worth repeating here. I mainly included this as an option for people who might change their queries to something more specific during development, but then later want to change back to sync everything and don't recall the syntax.